plaso/helpers/macos/dns.yaml

```diff
@@ -0,0 +1,155 @@
+# dtFabric format specification.
+---
+name: aul_dns
+type: format
+description: Apple Unified Logging and Activity Tracing formats
+urls: ["https://github.com/libyal/dtformats/blob/main/documentation/Apple%20Unified%20Logging%20and%20Activity%20Tracing%20formats.ascidoc"]
+---
+name: byte
+type: integer
+attributes:
+  format: unsigned
+  size: 1
+  units: bytes
+---
+name: char
+type: integer
+attributes:
+  format: signed
+  size: 1
+  units: bytes
+---
+name: uint8
+type: integer
+attributes:
+  format: unsigned
+  size: 1
+  units: bytes
+---
+name: uint8be
+type: integer
+attributes:
+  byte_order: big-endian
+  format: unsigned
+  size: 1
+  units: bytes
+---
+name: uint16be
+type: integer
+attributes:
+  byte_order: big-endian
+  format: unsigned
+  size: 2
+  units: bytes
+---
+name: uint32be
+type: integer
+attributes:
+  byte_order: big-endian
+  format: unsigned
+  size: 4
+  units: bytes
+---
+name: ipv4_address
+type: structure
+description: 4 ints representing octets
+attributes:
+  byte_order: little-endian
+members:
+- name: segments
+  type: sequence
+  element_data_type: uint8
+  number_of_elements: 4
+---
+name: ipv6_address
+type: structure
+description: 16 bytes
+attributes:
+  byte_order: little-endian
+members:
+- name: ip
+  type: stream
+  element_data_type: byte
+  elements_data_size: 16
+---
+name: dns_header
+type: structure
+description: DNS Header Data
+attributes:
+  byte_order: little-endian
```

Comment:
Why not make this `big-endian` instead of setting the byte order of each attribute individually?